CMSIS/NN/Source/ConvolutionFunctions/arm_depthwise_conv_s8_opt.c

```diff
@@ -22,7 +22,7 @@
  * Description:  Optimized s8 depthwise separable convolution function for
  *               channel multiplier of 1.
  *
- * $Date:        August 2019
+ * $Date:        January 2020
```

Comment:
please add date(new :-) ) as well..
and increment the copyright to 2020..

Rest looks good!